question-service/src/routes/questionsRoute.ts

```diff
@@ -61,8 +79,13 @@ router.post("/", async (req: Request, res: Response): Promise<Response> => {
         return res.status(200).send({
             message: "Question successfully created",
         });
-    } catch (error) {
+    } catch (error: any) {
         console.log(error);
+        if (error.code === 11000) {
```

Comment:
would be nice to identify what this error code is. as far as i can see this looks like a magic number

Reply:
gd point thanks! fixed